libs/Geometry3D/test/CLinkedVector3DTests.cpp

```diff
@@ -95,6 +95,54 @@ namespace Geometry3DCLinkedVector3DTests {
         EXPECT_DOUBLE_EQ(expected_len, linked_vector.getLen());
     }
 
+    TEST_F(CLinkedVector3DFixture, CLinkedVector3D_getPointBegin_Test) {
+        CPoint3D point_begin = nextRandomPoint3D();
+        CPoint3D point_end = nextRandomPoint3D();
+        CLinkedVector3D linked_vector(point_begin, point_end);
+
+        const CPoint3D& point_begin_copy = linked_vector.getPointBegin();
+
+        EXPECT_DOUBLE_EQ(point_begin.getX(), point_begin_copy.getX());
+        EXPECT_DOUBLE_EQ(point_begin.getY(), point_begin_copy.getY());
+        EXPECT_DOUBLE_EQ(point_begin.getZ(), point_begin_copy.getZ());
+    }
+
+    TEST_F(CLinkedVector3DFixture, CLinkedVector3D_getPointEnd_Test) {
+        CPoint3D point_begin = nextRandomPoint3D();
+        CPoint3D point_end = nextRandomPoint3D();
+        CLinkedVector3D linked_vector(point_begin, point_end);
+
+        const CPoint3D& point_end_copy = linked_vector.getPointEnd();
+
+        EXPECT_DOUBLE_EQ(point_end.getX(), point_end_copy.getX());
+        EXPECT_DOUBLE_EQ(point_end.getY(), point_end_copy.getY());
+        EXPECT_DOUBLE_EQ(point_end.getZ(), point_end_copy.getZ());
+    }
+
+    TEST_F(CLinkedVector3DFixture, CLinkedVector3D_setPointBegin_Test) {
```

Comment:
Same with memory addreses here, compare them, not values

Reply:
Complete